src/main/main.c

```diff
@@ -39,6 +39,8 @@ static	t_error_code	handle_valid_input(t_shell_op *sp)
 {
 	t_error_code	err;
 
+	if (0 == **(sp->input))
```

Comment:
i dont think this should be here
handle_valid_input() should only be called when needed.
so this check shoulod be in internal_flow()
something like:
`
if (NO_INPUT == err)
	continue ;
`